src/main/java/duke/Parser.java

```diff
@@ -20,15 +20,15 @@ else if (str.split(" ")[0].equals("unmark")) {
                     taskList.unmarkTask(str, taskList);
                     storage.writeToFile(taskList);
                 }
-                else if(str.split(" ", 2)[0].equals("duke.todo")) {
+                else if(str.split(" ", 2)[0].equals("todo")) {
```

Comment:
Small improvements you could consider! Free free to ignore they are mostly style related
1. Give a name to` str.split(" ", 2)[0]`, maybe call it header or something more meaningful
2.  Can consider using switch statement since you're checking for equality throughout